### DISCUSSION TITLE
Add Wiki Docs for debug w/ VS Code + STM32CubeCLT + cortex-debug extension

### DISCUSSION THREAD

**Discussion by bryceschober** [stm32duino/Arduino_Core_STM32#2574] [WiKi] [answered]
**Is your feature request/improvement related to a problem? Please describe.**

There are no documented suggestions for debugging with VS Code + STM32CubeCLT + cortex-debug extension.

**Describe the solution you'd like**

I figured out how to make it work, so I'd like to share. I wrote this page on the cortex-debug extension's wiki because it was publicly editable: https://github.com/Marus/cortex-debug/wiki/STM32CubeCLT-Specific-Configuration-(using-ST%E2%80%90Link)

I think it would be very helpful to add to it as a new "Visual Studio Code and STM32CubeCLT and cortex-debug extension" section on your <https://github.com/stm32duino/Arduino_Core_STM32/wiki/How-to-debug> page.

**Comment by fpistm** [comment]
Hi @bryceschober 
Thanks for the documentation.
You can follow this to contribute to the WiKi:
https://github.com/stm32duino/Arduino_Core_STM32/wiki/Update-the-WiKi

 I know it is not the easiest way but we had too much spam when it was public.

> [!NOTE]
> I've converted your issue as a discussion in the dedicated WiKi section.

**Reply by bryceschober** [reply]
@fpistm Hmm... the directions there aren't sufficiently clear. I did clone the wiki git repo and make my edits locally. I created a branch, but it didn't allow me to push it to the wiki repo. That made sense to me, and I thought, oh, I need to fork the repo and push my branch there... But GitHub forks don't also fork wikis... So I'm left with the question: Which github repo would I push my wiki branch to?

**Reply by fpistm** [reply]
Well, it is explicitly told:
`Duplicate the repository following this guide`

Anyway, I will try to provide another way...

**Reply by fpistm** [reply]
@bryceschober 
I've updated the way to propose change on the Wiki. See again this https://github.com/stm32duino/Arduino_Core_STM32/wiki/Update-the-WiKi

**Reply by bryceschober** [reply]
Thanks, your documentation updates, that is very clear now.

**Reply by bryceschober** [reply, marked as answer]
See https://github.com/stm32duino/wiki_contrib/pull/1
